i18n/en.json

```diff
@@ -2412,6 +2412,18 @@
     "translation": "[{{ .SiteName }}] Your password has been updated"
   },
   {
+    "id": "api.templates.user_access_token_body.info",
+    "translation": "A user access token was added to your account on {{ .SiteURL }}. They can be used by integrations to access {{.SiteName}} with your account.<br>If this change wasn't initiated by you, please contact your system administrator."
```

Comment:
Yeah, I was thinking about that too, but in most other help texts we say something along the lines of 

```
user access tokens ... can be used by integrations to ...
```

so I was staying consistent with it.

Could it be considered a (minor) security issue if the email says `can be used by integrations` when in fact it could be by anyone?